sos/plugins/pmem.py

```diff
@@ -0,0 +1,77 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin
+
+
+class Pmem(Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin):
+    """Persistent Memory devices
+    """
+
+    plugin_name = 'pmem'
+    profiles = ('storage', 'hardware', 'memory')
+
+    def setup(self):
+        self.add_cmd_output([
+            "ndctl --version",
+            "ndctl list -B",
+            "ndctl list -D",
+            "ndctl list -iD",
+            "ndctl list -iDF",
```

Comment:
Could you add 'H' option to ndctl list? For example, "ndctl list -iDFH". 'H' shows the DIMM health status.

Reply:
Added, thanks.